src/lib.rs

```diff
@@ -65,25 +65,40 @@ impl WakuLightNode {
     pub fn request_peers(&mut self, peer: &PeerId) {
         self.swarm.behaviour_mut().peer_exchange.send_request(
             peer,
-            peer_exchange::messages::PeerExchangeQuery { num_peers: 5 },
+            peer_exchange::messages::PeerExchangeRpc {
+                query: Some(peer_exchange::messages::PeerExchangeQuery { num_peers: 5 }),
+                response: None,
+            },
         );
     }
 
-    pub fn send_message(&mut self, peer: &PeerId, content_topic: String, payload: Vec<u8>) {
+    pub fn send_message(
+        &mut self,
+        peer: &PeerId,
+        content_topic: String,
+        payload: Vec<u8>,
+    ) -> Result<(), Error> {
+        // let timestamp = SystemTime::now()
+        //     .duration_since(UNIX_EPOCH)?
+        //     .as_secs()
+        //     .try_into()?;
         self.swarm.behaviour_mut().light_push.send_request(
             peer,
-            light_push::messages::PushRequest {
-                pubsub_topic: DEFAULT_PUBSUB_TOPIC.to_string(),
-                message: Some(light_push::message::WakuMessage {
-                    content_topic,
-                    payload,
-                    ephemeral: Some(false),
-                    meta: None,
-                    timestamp: None,
-                    version: None,
+            light_push::messages::PushRpc {
```

Comment:
Ah, so this is what we were doing wrong.  Strange that request and response are optional.  But now i think about it, with protobuf v3, all fields are optional - so im not sure why people sometimes anyway annotate them as optional.  Great that you figured this out!